Tasks/MavenAuthenticateV0/mavenauth.ts

```diff
@@ -25,7 +25,9 @@ async function run(): Promise<void> {
 
         let userM2FolderPath: string = "";
 
-        if (tl.osType().match(/^Win/)) {
+        if (tl.getInput("useAgentTempDir",false)) {
```

Comment:
There should be getBoolInput instead- otherwise it will always be true

Reply:
I also tested PR changes with amendment to this - LGTM!

Reply:
This is fixed in latest push.